packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -42,9 +42,11 @@ export class OdspDeltaStorageService {
             const storageToken = await this.getStorageToken(options, "DeltaStorage");
 
             const formBoundary = uuid();
+            const SPResponseGuid = uuid();
             let postBody = `--${formBoundary}\r\n`;
             postBody += `Authorization: Bearer ${storageToken}\r\n`;
             postBody += `X-HTTP-Method-Override: GET\r\n`;
+            postBody += `SPResponseGuid: ${SPResponseGuid}\r\n`;
```

Comment:
I'm pretty sure it has to have different name - it's provided by client, not spo.

Reply:
Yes I ran by him. The name is correct. The spo then uses this id and it becomes same as sprequestguid.